webapp/packages/plugin-authentication/src/Dialog/useAuthDialogState.ts

```diff
@@ -67,7 +67,7 @@ export function useAuthDialogState(accessRequest: boolean, providerId: string |
   const providers = authProvidersResource.data.filter(notEmptyProvider).sort(compareProviders);
 
   const activeProviders = providers.filter(provider => {
-    if (provider.federated || provider.trusted || provider.private) {
+    if (provider.federated || provider.trusted || provider.private || provider.authHidden) {
```

Comment:
why we can't use private? or trusted

Reply:
private does not fit here due to the fact we won't see the setting in the server config
trusted does not fit also here, but not sure why - the backend said so

we had a call with all of the flags which can fit to our needs, so we decided to extend the scheme cause there was no best candidate to fit our needs